project/Dependencies.scala

```diff
@@ -197,6 +197,16 @@ object Dependencies {
     "com.azure.resourcemanager" % "azure-resourcemanager" % "2.18.0"
   )
 
+  val wsmDependencies: List[ModuleID] = List(
+    "bio.terra" % "workspace-manager-client" % "0.254.452-SNAPSHOT"
+      exclude("jakarta.xml.bind", "jakarta.xml.bind-api")
+      exclude("jakarta.activation", "jakarta.activation-api")
+      exclude("jakarta.activation", "jakarta.activation")
+      exclude("jakarta.annotation", "jakarta.annotation-api")
+      exclude("com.sun.activation", "jakarta.activation")
+      exclude("org.glassfish.hk2.external", "jakarta.inject")
```

Comment:
I'm trying to get this to a point where it actually makes the REST call to WSM, and hit a problem I think it caused by the removal of this library.
```
java.lang.IllegalStateException: InjectionManagerFactory not found.
	at org.glassfish.jersey.internal.inject.Injections.lambda$lookupInjectionManagerFactory$0(Injections.java:74)
	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
	at org.glassfish.jersey.internal.inject.Injections.lookupInjectionManagerFactory(Injections.java:74)
```

Reply:
Is that in scope for this story or should we merge it and work on that on `develop` in WX-696?